rotkehlchen/tests/utils/history.py

```diff
@@ -293,34 +293,35 @@ def mock_binance_api_queries(url, params, *args, **kwargs):  # pylint: disable=u
             # Can't mock unknown assets in binance trade query since
             # only all known pairs are queried
             payload = '[]'
-            if params.get('symbol') == 'ETHBTC':
-                payload = """[{
-                "symbol": "ETHBTC",
-                "id": 1,
-                "orderId": 1,
-                "price": "0.0063213",
-                "qty": "5.0",
-                "commission": "0.005",
-                "commissionAsset": "ETH",
-                "time": 1512561941000,
-                "isBuyer": true,
-                "isMaker": false,
-                "isBestMatch": true
-                }]"""
-            elif params.get('symbol') == 'RDNETH':
-                payload = """[{
-                "symbol": "RDNETH",
-                "id": 2,
-                "orderId": 2,
-                "price": "0.0063213",
-                "qty": "5.0",
-                "commission": "0.005",
-                "commissionAsset": "RDN",
-                "time": 1512561942000,
-                "isBuyer": false,
-                "isMaker": false,
-                "isBestMatch": true
-                }]"""
+            if params['fromId'] == 0:
```

Comment:
why this extra check here?

Reply:
because the test queries it twice and the mock was desiged to always return a result. In the case of the test it expects that the second time nothing gets returned and for that we need to check the fromID